streams/uk.m3u

```diff
@@ -405,3 +405,9 @@ https://ve-uhd-push-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:uhd_stream_
 https://ve-uhd-push-uk-live.akamaized.net/x=3/i=urn:bbc:pips:service:uhd_stream_05/iptv_uhd_v1.mpd
 #EXTINF:-1 tvg-id="" status="error",Channel S (US Pacific) (720p) [Not 24/7]
 https://a.jsrdn.com/broadcast/22679_24MrQma9TX/-0800/c.m3u8
+#EXTINF:-1 tvg-id="" status="online",Shelanu TV (720p)
+https://1247634592.rsc.cdn77.org/1247634592/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="" status="online",Latest TV [Not 24/7]
+https://5a0e89631aa14.streamlock.net/LatestTV/LatestTV/playlist.m3u8
+#EXTINF:-1 tvg-id="" status="online",La Muscle TV (360p)
```

Comment:
This is not live

Reply:
Understanding that the name is "LA Muscle TV", they have WAAAAAAAAAAAY too much video links inside their player. I think i'll need to deduct it from the link we have.

Reply:
(and their "live TV" isn't really one. it's just a bunch of their vids put place in place to make it "look like it's live")

Reply:
![image](https://user-images.githubusercontent.com/30985701/185809988-e2bfac66-0593-4c88-8c0c-c94e1aca3c3d.png)